modules/fedimint-mint-client/src/lib.rs

```diff
@@ -758,6 +758,28 @@ impl MintClientModule {
             .await
     }
 
+    pub async fn get_notes_by_tier_counts(
```

Comment:
Docs would be nice, seems to return *up to* the number of notes for each denomination specified in `counts`?